testcontainers-dapr/src/main/java/io/dapr/testcontainers/MetadataEntry.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.testcontainers;
+
+public class MetadataEntry {
+  private String name;
+  private Object value;
```

Comment:
If you make value a string, you will not need the QuotedBoolean and you will comply to the CRD spec for Component because it is defined as a string there also.

Reply:
done